src/engine/renderer/Material.cpp

```diff
@@ -1104,6 +1105,107 @@ void BindShaderLightMapping( Material* material ) {
 	}
 	// FIXME: else
 
+	// bind u_LightGrid1
```

Comment:
Why is there a lot of new code here?

Reply:
Hmm, that can be removed since light/deluxe grid textures don't change and they're set always set elsewhere.

Reply:
I removed the same code from `UpdateSurfaceDataLightMapping()` because other shaders also set global uniforms in `BindShader*()`.